input/examples/Bundle-multiple-summ-report.json

```diff
@@ -417,7 +417,7 @@
             }
           ]
         },
-        "measure": "http://hl7.org/fhir/us/davinci-deqm/Measure/measure-exm130-example",
+        "measure": "http://hl7.org/fhir/us/davinci-deqm/Measure/measure-exm130-example|5.0.0-ballot",
```

Comment:
For measure-exm130-example (for all examples), the version used here should match what's defined in the measure-exm130-example of using 5.0.0-ballot or 5.0.0. 

  "url": "http://hl7.org/fhir/us/davinci-deqm/Measure/measure-exm130-example",
  "version": "2.0.0",